dkist/io/asdf/converters/tiled_dataset.py

```diff
@@ -5,16 +5,22 @@ class TiledDatasetConverter(Converter):
     tags = [
         "tag:dkist.nso.edu:dkist/tiled_dataset-0.1.0",
         "asdf://dkist.nso.edu/tags/tiled_dataset-1.0.0",
+        "asdf://dkist.nso.edu/tags/tiled_dataset-1.1.0",
     ]
     types = ["dkist.dataset.tiled_dataset.TiledDataset"]
 
     def from_yaml_tree(cls, node, tag, ctx):
         from dkist.dataset.tiled_dataset import TiledDataset
 
-        return TiledDataset(node["datasets"], node["inventory"])
+        return TiledDataset(node["datasets"], node["inventory"], meta=node.get("meta"))
 
     def to_yaml_tree(cls, tiled_dataset, tag, ctx):
         tree = {}
-        tree["inventory"] = tiled_dataset._inventory
+        meta = copy.deepcopy(tiled_dataset.meta)
+        inventory = meta.pop("inventory")
+        # If the history key has been injected into the meta, do not save it
+        meta.pop("history")
+        tree["inventory"] = inventory
```

Comment:
If I am bumping the schema version here anyway, I might as well move inventory into meta.